superset/models/core.py

```diff
@@ -207,6 +215,7 @@ def data(self) -> Dict[str, Any]:
             "id": self.id,
             "name": self.database_name,
             "backend": self.backend,
+            "configurationMethod": self.configuration_method,
```

Comment:
The columns are return in snake_case:

```suggestion
            "configuration_method": self.configuration_method,
```